platform/web/os_web.h

```diff
@@ -54,6 +54,11 @@ class OS_Web : public OS_Unix {
 	static void fs_sync_callback();
 	static void update_pwa_state_callback();
 
+	static int low_processor_usage_mode_get_callback(); 
```

Comment:
```suggestion
	static int low_processor_usage_mode_get_callback();
```

Reply:
Thanks! I didn't see that there was a linting error, thanks for being more alert than me, haha.

Reply:
They are so insidious these errors!